v2/python/README.md

```diff
@@ -0,0 +1,54 @@
+# AdSense Management API v2 Samples
+
+A collection of command-line samples for the AdSense Management API.
+
+##Installation and first request
+
+1. Download Google APIs Client Library for Python (google-api-python-client) and
+   the OAuth library
+   (https://github.com/googleapis/google-auth-library-python-oauthlib):
+  - https://code.google.com/p/google-api-python-client/
+  - https://github.com/googleapis/google-auth-library-python-oauthlib
+
+  or use pip:
+
+  ```bash
+  $ pip install google-api-python-client
+  $ pip install google_auth_oauthlib
+  ```
+
+2. (No longer needed, check on this) Make sure you can import the client library:
+
+  ```
+  $ python
+  >>> import apiclient
+  ```
+
+3. Execute any of the scripts to begin the auth flow:
+
+  ```bash
+  $ python get_all_accounts.py
+  ```
+
+  A browser window will open and ask you to login. Use the AdSense account.
+
+  Note: some examples require an argument to be passed in in order to run,
```

Comment:
Suggestion to add a note that "To determine what arguments are required you can pass in `help` as an argument, e.g., `python3 get_all_ad_units.py help`".